packages/services/src/kernel/messages.ts

```diff
@@ -153,6 +153,11 @@ export namespace KernelMessage {
     | 'shutdown_reply'
     | 'shutdown_request';
 
+  /**
+   * Control message types.
+   */
+  export type ControlMessageType = 'never';
```

Comment:
just `never`, not the string `'never'`

Reply:
https://www.typescriptlang.org/docs/handbook/basic-types.html#never

Reply:
Now I can ask github to add a "shame" emoji ;)